src/lib.rs

```diff
@@ -2,7 +2,7 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
-extern crate cairo_sys as ffi;
+pub extern crate cairo_sys as ffi;
```

Comment:
This bit seems to lack context. Generally the syntax is obsolete and it shouldn't be necessary.